### PR TITLE
Adding login

### PR DESCRIPTION
**Pull request by alexalghisi**
Screenshot 2019-06-06 at 10 02 35

### CODE REVIEW

File: src/components/HomeScreen.js [hunk 1]
```diff
@@ -21,26 +21,48 @@ export default class LoginView extends Component {
   }
 
   onClickListener = (viewId) => {
-    Alert.alert("Alert", "Button pressed "+viewId);
+    //Alert.alert("Alert", "Button pressed "+viewId);
```
Comment (Sfirleaalex) [inline src/components/HomeScreen.js]: Why is this commented out?

File: src/components/HomeScreen.js [hunk 2]
```diff
@@ -21,26 +21,48 @@ export default class LoginView extends Component {
...
+  };
+
+  validate = (val) => {
+    console.log(val);
+    let reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/ ;
+    if(reg.test(val) === false)
+    {
+    console.log("Email is Not Correct");
```
Comment (Sfirleaalex) [inline src/components/HomeScreen.js]: Client could not see logs please put allerts

File: src/components/HomeScreen.js (hunk 1 above)
Comment (alexalghisi) [reply src/components/HomeScreen.js]: Because it's not working properly

File: src/components/HomeScreen.js (hunk 2 above)
Comment (alexalghisi) [reply src/components/HomeScreen.js]: Aleert is not working properly 😢

File: src/components/HomeScreen.js [hunk 3]
```diff
@@ -20,38 +21,64 @@ export default class LoginView extends Component {
...
         <View style={styles.inputContainer}>
-          <TextInput style={styles.inputs}
+          <TextInput
+              style={styles.inputs}
               placeholder="Email"
               keyboardType="email-address"
               underlineColorAndroid='transparent'
-              onChangeText={(email) => this.setState({email})}/>
-        </View>
+              //onChangeText={text => this.setState({ email: text })}
```
Comment (Sfirleaalex) [inline src/components/HomeScreen.js]: Put line back is ok

File: src/components/HomeScreen.js [hunk 4]
```diff
@@ -20,38 +21,64 @@ export default class LoginView extends Component {
...
 
         <View style={styles.inputContainer}>
-          <TextInput style={styles.inputs}
+          <TextInput
+              style={styles.inputs}
               placeholder="Parola"
               secureTextEntry={true}
               underlineColorAndroid='transparent'
-              onChangeText={(password) => this.setState({password})}/>
+              //onChangeText={text => this.setState({ password: text })}
```
Comment (Sfirleaalex) [inline src/components/HomeScreen.js]: Same here

File: src/components/HomeScreen.js [hunk 5]
```diff
@@ -4,56 +4,79 @@ import {
...
+  verify() {
+    if(this.state.password.length<3) {
+      Alert.alert("password too short");
+      return;
+    }
+
+    if(this.validate(this.state.email))
+         this.props.navigation.navigate('WelcomeScreen');
+    else
+      Alert.alert("email wrong format")
```
Comment (Sfirleaalex) [inline src/components/HomeScreen.js]: `else {
Alert....
}`

File: src/components/HomeScreen.js [hunk 6]
```diff
@@ -4,56 +4,79 @@ import {
...
+
+  };
+
+  validate = (val) => {
+    let reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+    if(!reg.test(val))
```
Comment (Sfirleaalex) [inline src/components/HomeScreen.js]: return `reg.test(val)`

**Pull request merged**